content/docs/index.mdx

```diff
@@ -0,0 +1,4 @@
+---
+title: Raydiant Documentation
+excerpt: Sample excerpt
+---
```

Comment:
Looks like content layer [has a dedicated](https://www.contentlayer.dev/docs) page for `/docs` that's different to `/docs/getting-started`. The breadcrumb generator was failing as it requires this title to generate the first breadcrumb part.

<img width="905" alt="image" src="https://user-images.githubusercontent.com/1712435/174941867-c1041a14-a2b5-41ae-afcb-7b2173e99121.png">

<img width="693" alt="image" src="https://user-images.githubusercontent.com/1712435/174941981-a7e15dc8-541a-4598-8d06-77cf1839d39a.png">

Reply:
Sorry, should've mentioned that I've made sure it's not possible to visit `/docs` - logo now links to `docs/get-started` and we have a redirect setup for `/docs` -> `/docs/get-started`.

Reply:
Just seeing your comment now about removing the root breadcrumb. I'll look into that - agree it's a tad confusing having a Markdown doc that's not rendered anywhere.

Reply:
Yeah, kinda useless having the "Raydiant Documentation" breadcrumb on the...Raydiant documentation site lol. Good call out! Fixed here: https://github.com/mirainc/developer-docs/pull/11/commits/0381599b06ac5104e4874c53351ca7e67aae7c7a

Update the for each to `.map` too. Just wanted to make sure we have a consistent code style across all Raydiant apps.